views/administration/forms/form-classes-manage.hbs

```diff
@@ -89,6 +89,22 @@
         {{/if}}
     </div>
 
+    <div class="mb-3">
+        <h4>
+            Einverständnis manuell erklären
+        </h4>
+        <p>
+            Du kannst die Einverständniserklärungen für Deine Schüler auch in Papierform von den Eltern erhalten. Sobald Du alle Einverständniserklärungen erhalten
```

Comment:
Vorschlag:
Wenn die Einverständniserklärung von allen Schülern und Eltern in Papierform erfolgt ist, kann hier das Einverständnis für die ganze Klasse manuell erklälrt werden.